http/1.6/modules/ROOT/pages/index.adoc

```diff
@@ -3,46 +3,34 @@
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-
 Anypoint Connector for HTTP (HTTP Connector) enables you to declare HTTP servers that listen to
-requests and trigger flows, as well as HTTP clients that can communicate with any
-HTTP service.
-
-HTTP Connector effectively allows you to both
-expose and consume HTTP based APIs. For example, APIkit and RESTConnect both use
-the HTTP connector internally to work with APIs.
+requests and trigger flows, as well as HTTP clients that can communicate with any HTTP service. Additionally, the connector enables you to both expose and consume HTTP-based APIs. For example, APIkit and RESTConnect both use HTTP Connector internally to work with APIs.
 
 HTTP Connector also supports HTTPS (TLS) connectivity, provides easy ways to serve static
-resources and secure servers, and handles a number of client authentication schemes.
+resources and secure servers, and handles client authentication schemes.
 
-Release Notes: xref:release-notes::connector/connector-http.adoc[HTTP Connector Release Notes] +
-Exchange: https://www.mulesoft.com/exchange/org.mule.connectors/mule-http-connector/[HTTP Connector]
+For information about compatibility and fixed issues, see the HTTP Connector Release Notes.
 
-== POM File Information
+== Prerequisites
 
-[source,xml,linenums]
-----
-<dependency>
-  <groupId>org.mule.connectors</groupId>
-  <artifactId>mule-http-connector</artifactId>
-  <version>x.x.x</version>
-  <classifier>mule-plugin</classifier>
-</dependency>
-----
+To use this connector, you must be familiar with:
+
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* How to create a Mule app using Anypoint Studio
+
+Before creating an app, you must have access to Anypoint Platform and Anypoint Studio
 
-Replace `x.x.x` with the version that corresponds to the connector you are using.
 
 == HTTP Servers
 
-HTTP servers expose services for users or applications to consume. They
-receive data in the form of requests and return a response. For the connector,
-this means triggering flows when a request is received and responding once the
-flow finishes. This is all handled by the HTTP Listener, a source the connector provides.
+HTTP servers expose services for users or applications to consume. These servers receive data in the form of requests and return a response. The HTTP *Listener* source triggers flows when a request is received and responds after the flow finishes.
```

Comment:
These servers receive data in the form of requests and return corresponding responses
or
Each server receives data in the form of a request and returns a corresponding response